example/sample1/federatedingress-template.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: core.federation.k8s.io/v1alpha1
+kind: FederatedIngress
+metadata:
+  name: test-ingress
+  namespace: test-namespace
+spec:
+  template:
+    spec:
+      rules:
+      - host: ingress.example.com
```

Comment:
Should here be `ing.example.com`?

Reply:
Good catch. Yes. I'll update the PR.

Reply:
I am going to update the .md to use ingress instead of ing so the yaml will be correct.